src/core/config/styles.js

```diff
@@ -74,7 +84,50 @@ export function getCurrentTheme(isContrastMode) {
                 root: {
                     border: "1px solid #e5e5e5"
                 }
-            }
+            },
+            MuiList: {
+                root: {
+                    backgroundColor: "#fff",
+                }
+            },
+            MuiPaper: {
+                elevation1: {
+                    boxShadow: "none",
+                    backgroundColor: "#fafafa",
```

Comment:
why aren't you using the palette colors? There should be no hardcoded colors outside of the palette

Reply:
Done

Reply:
Done